ontotext-yasgui-web-component/src/components/ontotext-yasgui-web-component/ontotext-yasgui-web-component.tsx

```diff
@@ -667,13 +669,14 @@ export class OntotextYasguiWebComponent {
 
   @Listen('internalQueryEvent')
   onQuery(event: CustomEvent<InternalQueryEvent>): void {
-    this.changeRenderMode(this.defaultViewMode);
-    this.defaultViewMode = RenderingMode.YASGUI;
     this.output.emit(toOutputEvent(event));
     this.getOntotextYasgui()
       .then((ontotextYasgui) => {
         ontotextYasgui.leaveFullScreen();
-        VisualisationUtils.setYasqeFullHeight(this.renderingMode, VisualisationUtils.resolveOrientation(this.isVerticalOrientation));
+        const editorHeight = ontotextYasgui.getTab(ontotextYasgui.getTabId()).persistentJson.yasqe.editorHeight;
+        this.changeRenderMode(this.defaultViewMode, editorHeight);
+        this.renderingMode = RenderingMode.YASGUI;
+        ontotextYasgui.refresh();
```

Comment:
Can we avoid this refresh?

Reply:
Yes, I removed it. Seems to work without it.